src/lib/models/codeObject.js

```diff
@@ -342,6 +342,6 @@ export default class CodeObject {
   }
 
   get fqid() {
-    return encodeURI(`${this.type}:${this.id}`);
+    return `${this.type}:${this.id}`;
```

Comment:
Let the library be agnostic to the encoding. This will prevent us from having to encode/decode elsewhere.